apps/wallet/src/ui/app/hooks/useGetNFTMeta.ts

```diff
@@ -20,7 +20,8 @@ export function useGetNFTMeta(objectID: string): NFTMetadata | null {
 
         const { details } = data || {};
         if (!isSuiObject(details) || !data) return null;
-        const fields = getObjectFields(data);
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const fields = getObjectFields(data) as Record<string, any> | undefined;
```

Comment:
Why do you need this casting? Shouldn't it still be a record just with refined types?

Reply:
`nftMeta`'s url field was being marked as `MoveValue` which was throwing a type error on the return statement. I've casted the url field as string instead. Good catch! 
![image](https://user-images.githubusercontent.com/7824233/210337507-56a5b379-296b-478a-a4bd-f9533c8fa33d.png)

Reply:
Yeah the casting at the value makes a lot more sense, ideally we'd actually do `typeof` checks in the product to ensure the value is what we expect, but this is fine for now.